src/Form/Form.js

```diff
@@ -1,21 +1,37 @@
 const FormAnalyzer = require('./FormAnalyzer')
-const {SUBMIT_BUTTON_SELECTOR, FORM_ELS_SELECTOR} = require('./selectors')
 const {addInlineStyles, removeInlineStyles, setValue, isEventWithinDax, isMobileApp, isApp} = require('../autofill-utils')
-const {getInputSubtype, setInputType, getInputMainType} = require('./input-classifiers')
+const {getInputSubtype, getInputMainType} = require('./matching')
 const {getIconStylesAutofilled, getIconStylesBase} = require('./inputStyles')
 const {ATTR_AUTOFILL} = require('../constants')
 const getInputConfig = require('./inputTypeConfig.js')
 const {getUnifiedExpiryDate, formatCCYear, getCountryName} = require('./formatters')
+const {Matching} = require('./matching')
+const {matchingConfiguration} = require('./matching-configuration')
 
 class Form {
-    constructor (form, input, deviceInterface) {
+    /**
+     * @type {import("./matching").Matching}
+     */
```

Comment:
Is it worth using the single-line comments like `/** @type {import("./matching").Matching} */` in all similar instances?

Reply:
done